frontend/workflows/experimentation/src/view-experiment-run.tsx

```diff
@@ -18,7 +19,7 @@ const ViewExperimentRun: React.FC<BaseWorkflowProps> = ({ heading }) => {
   const [experiment, setExperiment] = useState<
     IClutch.chaos.experimentation.v1.ExperimentRunDetails | undefined
   >(undefined);
-  const [error, setError] = useState("");
+  const [error, setError] = useState(undefined);
```

Comment:
+1

Reply:
Same as above: Can/should we specify the type to be `ClutchError | undefined` here?